<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagePage</class>
 <widget class="QWidget" name="MessagePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="verticalFrame">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;sss</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9"/>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="text">
      <string>These are your sent and received encrypted messages. Click on an item to read it.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Click on a message to view it</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="messageDetails">
       <property name="title">
        <string>Message Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelType">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="typeLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelContact">
              <property name="text">
               <string>Contact:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="contactLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="messageLabel">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="message">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="replyButton">
         <property name="toolTip">
          <string>Sign a message to prove you own a Bitcointalkcoin address</string>
         </property>
         <property name="text">
          <string>&amp;Reply</string>
         </property>
         <property name="icon">
          <iconset resource="../bitcointalkcoin.qrc">
           <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyFromAddressButton">
         <property name="toolTip">
          <string>Copy the currently selected address to the system clipboard</string>
         </property>
         <property name="text">
          <string>&amp;Copy From Address</string>
         </property>
         <property name="icon">
          <iconset resource="../bitcointalkcoin.qrc">
           <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyToAddressButton">
         <property name="toolTip">
          <string>Copy the currently selected address to the system clipboard</string>
         </property>
         <property name="text">
          <string>Copy To &amp;Address</string>
         </property>
         <property name="icon">
          <iconset resource="../bitcointalkcoin.qrc">
           <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="toolTip">
          <string>Delete the currently selected address from the list</string>
         </property>
         <property name="text">
          <string>&amp;Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../bitcointalkcoin.qrc">
           <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcointalkcoin.qrc"/>
 </resources>
 <connections/>
</ui>
